File: collect/jvmcover/testdata/sample/src/test/java/demo/CalculatorTest.java
package demo;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

class CalculatorTest {
    @Test
    void testAdd() {
        assertEquals(5, Calculator.add(2, 3));
    }

    @Test
    void testSub() {
        assertEquals(1, Calculator.sub(3, 2));
    }
}

File: collect/phpcover/testdata/sample/tests/CalculatorTest.php
<?php

declare(strict_types=1);

namespace Demo\Tests;

use Demo\Calculator;
use PHPUnit\Framework\TestCase;

final class CalculatorTest extends TestCase
{
    public function testAdd(): void
    {
        $this->assertSame(5, Calculator::add(2, 3));
    }

    public function testSub(): void
    {
        $this->assertSame(1, Calculator::sub(3, 2));
    }
}
